appinventor/blocklyeditor/src/warningHandler.js

```diff
@@ -435,7 +423,7 @@ Blockly.WarningHandler.prototype['checkDropDownContainsValidValue'] = function(b
         textInDropDown = true;
         // A mismatch in the untranslated value and translated text can be corrected.
         if (dropDownList[k][0] != text) {
-          dropDown.setText(dropDownList[k][0]);
+          // dropDown.setText(dropDownList[k][0]);
```

Comment:
This is from commit 97022336749, and I'm not sure that it should actually be commented out in the long run. Need to verify.

Reply:
I think that that commented out code can just be replaced by:

```
dropDown.setValue(dropDownList[k][0]);
```